models/confirmation.go

```diff
@@ -68,6 +70,9 @@ const (
 	TypePatientPasswordReset Type = "patient_password_reset"
 	TypePatientPasswordInfo  Type = "patient_password_info"
 	TypeCareteamInvite       Type = "careteam_invitation"
+	TypeMedicalTeamInvite    Type = "medicalteam_invitation"
```

Comment:
is it possible to add a comment to indicates the differences between careteam and medicalteam?